.gitignore

```diff
@@ -1 +1,7 @@
+.*
```

Comment:
Could you, please, explain, why .* added?

Reply:
We have two options: add all hidden files to .gitignore (eg .idea) or ignore all hidden files except those we want to commit (eg .dockerignore). I prefer the second option.

Reply:
> We have two options

No, we have more than these two. Contributor can add his IDEs and other tools files to his global gitignore, with command like `git config --global core.excludesfile '~/.gitignore'`. So I prefer this. Also I prefer not to do mix non-relevant changes in functional commits, so it would be better to revert these changes.